tests/integration/link/test_link_children.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import dash
+from dash.dependencies import Input, Output
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+@pytest.mark.DCC776
+def test_lich001_default(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [
+            dcc.Link(id="link1", href="/page-1"),
+            dcc.Location(id="url", refresh=False),
+            html.Div(id="content")
+        ]
+    )
+    @app.callback(Output("content", "children"), [Input("link1", "children")])
```

Comment:
Does this callback and `content` Div do anything in this test?

Reply:
Can you add another simple test checking that children are indeed children when defined

Reply:
Added in 5eac66e.